packages/perspective-workspace/src/ts/utils/observable_map.ts

```diff
@@ -25,12 +25,8 @@ export class ObservableMap<K, V> extends Map<K, V> {
     }
 
     delete(name: K) {
-        const result = this._delete_listener?.(name);
```

Comment:
The delete listener registered by the workspace always returns `undefined` now, so this ended up always taking the `else` branch below, which skips removing the key from the map.  It looks like the delete listener is not meant anymore to prevent deletes, so I removed the conditional entirely.